backend/ohq/statistics.py

```diff
@@ -232,3 +233,69 @@ def queue_calculate_questions_per_ta_heatmap(queue, weekday, hour):
         hour=hour,
         defaults={"value": statistic if statistic else 0},
     )
+
+
+def user_calculate_questions_asked(user):
+    num_questions = Decimal(Question.objects.filter(asked_by=user).count())
+    UserStatistic.objects.update_or_create(
+        user=user,
+        metric=UserStatistic.METRIC_TOTAL_QUESTIONS_ASKED,
+        defaults={"value": num_questions},
+    )
+
+
+def user_calculate_questions_answered(user):
+    num_questions = Decimal(Question.objects.filter(responded_to_by=user).count())
+    UserStatistic.objects.update_or_create(
+        user=user,
+        metric=UserStatistic.METRIC_TOTAL_QUESTIONS_ANSWERED,
+        defaults={"value": num_questions},
+    )
+
+
+
+def user_calculate_time_helped(user):
+    user_time_helped = (
+        Question.objects.filter(asked_by=user)
+        .aggregate(time_helped=Sum(F("time_responded_to") - F("time_response_started")))
+    )
+    time = user_time_helped["time_helped"]
+
+    UserStatistic.objects.update_or_create(
+        user=user,
+        metric=UserStatistic.METRIC_TOTAL_TIME_BEING_HELPED,
+        defaults={"value": time.seconds if time else Decimal('0.00000000')},
```

Comment:
I am more inclined to not call `update_or_create` at all if `time.seconds` is `None` instead of updating the value to be 0.